Files/Helpers/StorageItemHelpers.cs

```diff
@@ -30,39 +31,49 @@ public static async Task<IStorageItem> ToStorageItem(this IStorageItemWithPath i
                 // for now, we can't
                 return default;
             }
-            else if (typeof(IStorageFile).IsAssignableFrom(typeof(TOut)))
+
+            // Fast get attributes
+            bool exists = NativeFileOperationsHelper.GetFileAttributesExFromApp(path, NativeFileOperationsHelper.GET_FILEEX_INFO_LEVELS.GetFileExInfoStandard, out NativeFileOperationsHelper.WIN32_FILE_ATTRIBUTE_DATA itemAttributes);
```

Comment:
I'm afraid this might fail on: phone storage, networks shares, zip files, ftp.. could you check?

Reply:
Indeed it does, I'll try to workaround it

Reply:
@gave92 Do you have any win32 function we could use to determine the file type? I'm out of ideas

Reply:
My suggestion is you do the reverse: _if_ GetFileAttributesExFromApp _succeedes_ then you use it to check if it's a file or folder, if id does not succeed keep current behaviour. It will still be faster in 90% of cases.

Reply:
Seems like a good idea 💡